test/chain-specific-spokepools/Polygon_SpokePool.ts

```diff
@@ -163,9 +165,10 @@ describe("Polygon Spoke Pool", function () {
   });
 
   it("PolygonTokenBridger retrieves and unwraps tokens correctly", async function () {
+    const chainId = await owner.getChainId();
     const polygonTokenBridger = await (
       await getContractFactory("PolygonTokenBridger", owner)
-    ).deploy(hubPool.address, weth.address);
+    ).deploy(hubPool.address, weth.address, chainId, chainId);
```

Comment:
I think you could test this by deploying new `PolygonTokenBridgers` with incorrect l1/l2 chainIDs and just testing that `send` and `retrieve` revert if the chain IDs are set unexpectedly

Reply:
Agree, will add a test for this!

Reply:
Done.